### ISSUE TITLE
Add seamless background parallax effect

### ISSUE THREAD

**Issue by glennposadas** [ealeksandrov/EAIntroView#202]
The **parallax** feature of your EAIntroView I believe is not documented? Also, when you say **parallax** in EAIntroView, it's the iOS 7's parallax and not the _**scrolling parallax**_. Are we going to have this in the future? Thanks! G.,

**Comment by ealeksandrov** [comment]
Right, that's smaller iOS background parallax.
This is interesting feature, but how do you want it to work? To use 1 big background image for all pages at once? And scroll it slower than front pages?

**Comment by glennposadas** [comment]
Yes, 1 large background image for all pages and scroll it slower than the front pages would be super cool for this library.

**Comment by ealeksandrov** [comment]
Ok, I'll keep this issue open for reference.
